spec/mailers/user_mailer_spec.rb

```diff
@@ -76,6 +76,15 @@
         "[Reservations] #{@res.equipment_model.name} Request Approved")
     end
 
+    it 'sends expired request notifications' do
+      @res.update_attributes(status: 'denied',
+                             flags: (Reservation::FLAGS[:request] |
```

Comment:
As a side-note, we should expand the flag "API" to implement a method that assigns multiple flags simultaneously using the existing `flag` method, but that's beyond scope for this issue (I'll create a new one).